addons/website/static/tests/tours/website_form_editor.js

```diff
@@ -100,10 +111,11 @@ odoo.define('website.tour.form_editor', function (require) {
         return ret;
     };
     const addCustomField = function (name, type, label, required, display) {
-        return addField(`data-custom-field="${name}"`, name, type, label, required, display);
+        return addField(`data-custom-field="${name}"`, name, type, label, required, true, display);
```

Comment:
```suggestion
        return addField(name, type, label, required, true, display);
```
Now that you actually have a boolean to know if this is a custom field or not, you could do this `data-custom-field="${name}"` vs `data-existing-field="${name}"` in the `addField` directly